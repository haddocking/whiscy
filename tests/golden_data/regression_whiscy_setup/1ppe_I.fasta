>1ppe_I.fasta
RVCPRILMECKKDSDCLAECVCLEHGYCG
