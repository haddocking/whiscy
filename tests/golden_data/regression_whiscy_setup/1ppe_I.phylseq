52  29
MASTER    RVCPRILMECKKDSDCLAECVCLEHGYCG
ITR1_CUCMARVCPRILMECKKDSDCLAECVCLEHGYCG
ITR4_CUCMARVCPRILMKCKKDSDCLAECVCLEHGYCG
ITR3_CUCPERVCPKILMECKKDSDCLAECICLEHGYCG
ITI1_LAGSIRRCPRIYMECKHDSDCLADCVCLEHGICG
ITR6_CYCPERICPRILMKCKKDSDCLAECICEEHGFCG
ITR7_CYCPERICPRILMKCKKDSDCLAECICQEHGFCG
ITR1_LUFAERICPRILMECSSDSDCLAECICLEQGFCG
ITR1_MOMRE-ICPRILMECKRDSDCLAQCVCKRQGYCG
ITR3_LUFAERICPRILMECSSDSDCLAECICLENGFCG
ITR2_LUFAERICPRILMECSSDSDCLAECICLEqgFCG
A0A0A0KV23-MCPRILMKCKHDSDCLPGCVCLEheYCG
ITR1_CITLARRCPRIYMECKRDADCLADCVCLQHGICG
ITR2B_CUCS-VCPKILMKCKHDSDCLLDCVCLEdgYCG
ITR2_BRYDIRGCPRILMRCKRDSDCLAGCVCQKNGYCG
ITR3_CYCPERICPRILMECKADSDCLAQCICEESGFCG
ITR4_CUCSA-MCPRILMKCKHDSDCLPGCVCLEheYCG
ITR4_CYCPERICPRILMECKADSDCLAQCICQENGFCG
ITR5_CYCPERICPRILMECKADSDCLAQCICQESGFCG
ITR4_LUFAE-ICPRILMPCSSDSDCLAECICLENGFCG
ITR2_ECBEL--CPRILMRCKQDSDCLAGCVCGPNGFCG
ITR2_SECEDRKCPKILMRCKRDSDCLAKCTCQESGYCG
ITR3_CUCMCRMCPKILMKCKQDSDCLLDCVCLKEGFCG
Q9S8W2_CUCRMCPKILMKCKQDSDCLLDCVCLKEGFCG
Q9S8W3_CUC-MCPKILMKCKQDSDCLLDCVCLKEGFCG
A0A0A0KY34--CPRILMKCKTDSDCYPGCTCKPMGYCG
ITR1_TRIKI--CPRILMPCKVNDDCLRGCKCLSNGYCG
Q9S8D2_CUC--CPRILMKCKTDRDCLTGCTCKRNGYCG
ITR1_MOMCHRRCPRILKQCKRDSDCPGECICMAHGFCG
ITR3_MOMCHRGCPRILKQCKQDSDCPGECICMAHGFCG
ITR3_MOMCORACPRILKKCRRDSDCPGECICKENGYCG
ITR5_LUFAE-ICPRILMPCKTDDDCMLDCRCLSNGYCG
A0A0A7HF87RACPRILKKCRRDSDCPGECICQGNGYCG
A0A0A7HF94RACPRILKRCRRDSDCPGACVCQDNGYCG
A0A0A7HIA5RACPRILKKCRRDSDCPGECICKGNGYCG
A0A0A7HIS7RACPRILKKCRRDSDCPGECICKGNGYCG
A0A0A7HIT2RACPRILKRCRRDSDCPGACVCQDNGYCG
A6N2U8_MOMRICPRIWMECKRDSDCMAQCICVDG-HCG
IELI_MOMCHRICPLIWMECKRDSDCLAQCICVDG-HCG
ITR2_MOMCHRICPRIWMECKRDSDCMAQCICVDG-HCG
J3R2I9_MOMRACPRILKKCRRDSDCPGECICKGNGYCG
J3R9Z5_MOMRACPRILKKCRRDSDCPGECICQGNGYCG
J3RCD6_MOMRACPRILKKCRRDSDCPGECICKGNGYCG
J7IN40_9RORACPRILKKCRRDSDCPGECICQGNGYCG
A0A0A0KT01RMCPRILEECTTDDDCMNDCICLSNGFCG
A0A0A7HF33RACPKILKRCRRDSDCPGACVCQDNGYCG
A0A0A7HG47RACPKILKRCRRDSDCPGACVCQDNGYCG
A0A0A7HIA9RACPKILKRCRRDSDCPGACVCQDNGYCG
ITR5_SECED--CPRILMKCKLDTDCFPTCTCRPSGFCG
ITRA_MOMCHRSCPRIWMECTRDSDCMAKCICVAG-HCG
ITR1_MOMCO-VCPKILQRCRRDSDCPGACICRGNGYCG
ITR2_MOMCO-VCPKILKKCRRDSDCPGACICRGNGYCG
