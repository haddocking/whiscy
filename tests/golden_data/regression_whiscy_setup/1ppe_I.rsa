REM  FreeSASA 2.0.3
REM  Absolute and relative SASAs for 1ppe_I.pdb
REM  Atomic radii and reference values for relative SASA: NACCESS
REM  Chains: I
REM  Algorithm: Lee & Richards
REM  Probe-radius: 1.40
REM  Slices: 20
REM RES _ NUM      All-atoms   Total-Side   Main-Chain    Non-polar    All polar
REM                ABS   REL    ABS   REL    ABS   REL    ABS   REL    ABS   REL
RES ARG I   1   171.08  71.8  95.93  48.8  75.15 180.1  70.35  91.1 100.73  62.5
RES VAL I   2   103.37  68.3  99.84  90.6   3.54   8.6 103.37  89.7   0.00   0.0
RES CYS I   3    33.39  24.9  15.33  16.6  18.06  43.1  16.88  17.3  16.51  45.2
RES PRO I   4    48.37  35.6  34.19  31.4  14.17  52.3  45.46  37.7   2.91  19.2
RES ARG I   5   244.01 102.4 200.27 101.9  43.73 104.8  90.16 116.7 153.85  95.5
RES ILE I   6    93.76  53.5  75.13  56.1  18.63  45.3  84.34  60.7   9.42  26.1
RES LEU I   7   141.42  79.3 113.80  81.9  27.63  69.9 117.13  82.5  24.29  66.6
RES MET I   8    43.96  22.7  40.18  26.4   3.78   9.1  43.92  27.9   0.04   0.1
RES GLU I   9   140.37  81.6 123.33  94.6  17.04  40.8  44.27  73.8  96.11  85.7
RES CYS I  10    13.03   9.7   0.00   0.0  13.03  31.1   7.46   7.6   5.58  15.3
RES LYS I  11   190.17  95.0 156.29  98.6  33.88  81.2 115.70  99.8  74.47  88.3
RES LYS I  12   114.68  57.3 110.92  70.0   3.76   9.0  97.25  83.9  17.43  20.7
RES ASP I  13    83.10  59.2  73.99  74.9   9.11  21.8  30.74  62.4  52.36  57.4
RES SER I  14    89.11  76.4  51.49  70.4  37.62  86.7  60.61 124.9  28.50  41.9
RES ASP I  15    59.10  42.1  35.82  36.3  23.27  55.7  22.57  45.8  36.52  40.1
RES CYS I  16    17.38  12.9   0.05   0.1  17.33  41.3   0.97   1.0  16.41  45.0
RES LEU I  17    84.34  47.3  82.51  59.4   1.83   4.6  83.61  58.9   0.74   2.0
RES ALA I  18    71.15  65.9  18.71  29.3  52.43 119.3  36.35  51.1  34.79  94.8
RES GLU I  19    93.16  54.1  86.04  66.0   7.12  17.1  50.82  84.8  42.33  37.8
RES CYS I  20     8.70   6.5   2.24   2.4   6.46  15.4   2.24   2.3   6.46  17.7
RES VAL I  21    55.59  36.7  55.59  50.4   0.00   0.0  55.59  48.2   0.00   0.0
RES CYS I  22    23.44  17.5   6.96   7.5  16.48  39.3   6.96   7.1  16.48  45.2
RES LEU I  23    92.69  52.0  86.84  62.5   5.85  14.8  89.59  63.1   3.09   8.5
RES GLU I  24   159.66  92.8 117.92  90.5  41.74 100.0  65.03 108.5  94.63  84.4
RES HIS I  25   152.58  83.5 128.82  89.5  23.76  61.3  96.99  99.8  55.59  64.9
RES GLY I  26    24.51  30.5   0.00   N/A  24.51  30.5  21.60  57.3   2.91   6.8
RES TYR I  27    97.27  45.8  96.78  55.6   0.49   1.3  57.79  42.5  39.48  51.7
RES CYS I  28    19.30  14.4   2.80   3.0  16.49  39.3   3.70   3.8  15.60  42.7
RES GLY I  29    33.89  42.2   0.00   N/A  33.89  42.2  17.10  45.4  16.78  39.4
END  Absolute sums over single chains surface
CHAIN  1 I     2502.6       1911.8        590.8       1538.6        964.0
END  Absolute sums over all chains
TOTAL          2502.6       1911.8        590.8       1538.6        964.0
