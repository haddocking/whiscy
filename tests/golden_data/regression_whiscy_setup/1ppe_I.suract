1
2
3
4
5
6
7
9
11
12
13
14
15
16
17
18
19
21
23
24
25
27
29
