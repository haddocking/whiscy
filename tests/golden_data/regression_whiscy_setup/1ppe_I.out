   52
MASTER         0.00000  0.00000  0.04347  0.08404  0.26776  0.23143  0.22248  0.26567  0.33996  0.26316  0.26567  0.34757  0.35751  0.34186  0.40905  0.35895  0.34757  0.31898  0.34525  0.34101  0.46614  0.50751  0.51518  0.51518  0.54172  0.69219  0.70405  0.63179  0.50578  0.50517  0.51631  0.60417  0.58135  0.61184  0.60595  0.60595  0.61184  0.50890  0.54595  0.50890  0.60595  0.58135  0.60595  0.58135  0.69730  0.67527  0.67527  0.67527  0.84038  0.65392  0.78076  0.72105
